aptos-move/aptos-aggregator/src/tests/types.rs

```diff
@@ -31,7 +31,8 @@ impl Default for FakeAggregatorView {
         Self {
             v1_store: HashMap::new(),
             v2_store: HashMap::new(),
-            counter: RefCell::new(0),
+            // Put some recognizable number, to easily spot missed exchanges
+            counter: RefCell::new(87654321),
```

Comment:
using something that ends in 0s might be more helpful to track down on which value the exchange was incorrect?

Reply:
I am doing that for actual prod (ends with 1000000), for these fake I put something distinct from that